saleor/product/migrations/0141_merge_20210212_1037.py

```diff
@@ -0,0 +1,13 @@
+# Generated by Django 3.1.5 on 2021-02-12 10:37
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("product", "0139_description_vector_search"),
+        ("product", "0140_auto_20210125_0905"),
+    ]
```

Comment:
Why is this migration needed?

Reply:
`0139_description_vector_search` is related to 2.11 migrations which are creating search vector, so we need to change it dependency and that change requires merge migration